assets/src/web-stories-block/block/components/storiesInspectorControls.js

```diff
@@ -0,0 +1,254 @@
+} from '@wordpress/components';
+import { InspectorControls } from '@wordpress/block-editor';
+import { RawHTML, useEffect } from '@wordpress/element';
+import { select } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { useConfig } from '../../../dashboard/app/config';
+import {
+  CAROUSEL_VIEW_TYPE,
+  CIRCLES_VIEW_TYPE,
+  GRID_VIEW_TYPE,
+  ORDER_BY_OPTIONS,
+} from '../constants';
+import { isShowing } from '../util';
+import AuthorSelection from './authorSelection';
+
+/**
+ * StoriesInspectorControls props.
+ *
+ * @typedef StoriesInspectorControlsProps
+ *
+ * @property {string} viewType String indicator of active view type.
+ * @property {number} numOfStories Number indicator of maximum number of stories to show.
+ * @property {number} numOfColumns Number indicator of number of columns in grid view type.
+ * @property {string} orderByValue String indicator of stories sorting.
+ * @property {boolean} isShowingTitle Whether or not to display story's title.
+ * @property {boolean} isShowingDate Whether or not to display story's date.
+ * @property {boolean} isShowingAuthor Whether or not to display story's author.
+ * @property {boolean} isShowingViewAll Whether or not to display stories archive link.
+ * @property {string} viewAllLinkLabel Archive link's label.
+ * @property {boolean} imageOnRight Whether or not to display images on right side in list view type.
+ * @property {Array} authors An array of authors objects which are currently selected.
+ * @property {Function} setAttributes Callable function for saving attribute values.
+ */
+
+/**
+ * LatestStoriesBlockControls component. Used for rendering block controls of the block.
+ *
+ * @param {StoriesInspectorControlsProps} props Component props.
+ *
+ * @return {*} JSX markup.
+ */
+const StoriesInspectorControls = (props) => {
+  const {
+    attributes: {
+      viewType,
+      numOfStories,
+      numOfColumns,
+      orderByValue,
+      viewAllLinkLabel,
+      authors,
+      sizeOfCircles,
+      fieldState,
+    },
+    setAttributes,
+    showFilters = true,
+  } = props;
+
+  const { fieldStates } = useConfig();
+
+  useEffect(() => {
+    // Set default field state on load.
+    if (!Object.entries(fieldState).length) {
+      setAttributes({
+        fieldState: fieldStates,
+      });
+    }
+  }, [fieldState, fieldStates, setAttributes]);
+
+  // Set up sort options.
+  const orderByOptions = Object.entries(ORDER_BY_OPTIONS).map(
+    ([key, option]) => {
+      return {
+        label: option.label,
+        value: key,
+      };
+    }
+  );
+
+  const previewLink = select('core/editor').getEditedPostPreviewLink();
+  const carouselMessage = sprintf(
+    /* Translators: Carousel informational message. 1: Preview link. */
+    __(
+      `<i><b>Note:</b> Carousel view's functionality will not work in Editor. <a target="__blank" href="%1$s">Preview</a> post to see it in action.</i>`,
+      'web-stories'
+    ),
+    previewLink
+  );
```

Comment:
Two things:

1. The `<i>` should be _outside_ the string.
2. Should use [`createInterpolateElement`](https://make.wordpress.org/core/2020/07/17/introducing-createinterpolateelement/) for using markup here, **not** `RawHTML`

So along these lines:

```js
const carouselMessage = createInterpolateElement(
    __(
      `<b>Note:</b> Carousel view's functionality will not work in Editor. <a>Preview</a> post to see it in action.</i>`,
      'web-stories'
    ),
    {
      b: <b />,
      a: <a href={previewLink} target="__blank" rel="noreferrer" />
    }
  );
```

And then below:

```js
<Notice
  className="web-stories-carousel-message"
  isDismissible={false}
  status="warning"
>
  {carouselMessage}
</Notice>
```

Aside: if Glider.js works in the editor we can remove this I think.

Reply:
I will update the notice change, but for the carousel in editor, I can look into it once the #201 gets merged.

Reply:
Don't have to completely remove `<i></i>` necessarily. But it must be _outside_ the translatable string.

```js
const carouselMessage = createInterpolateElement(
    __(
      `<b>Note:</b> Carousel view's functionality will not work in Editor. <a>Preview</a> post to see it in action.</i>`,
      'web-stories'
    ),
    {
      b: <b />,
      a: <a href={previewLink} target="__blank" rel="noreferrer" />
    }
  );



<Notice
  className="web-stories-carousel-message"
  isDismissible={false}
  status="warning"
>
  <i>{carouselMessage}</i>
</Notice>
```

Reply:
> The notice text should be clearly and easily readable.

Aside: `<i></i>` actually does the opposite for people with different accessibility needs, but let's not get into details :-)

Reply:
I was not aware about that surely something to learn about. since the text is small with slant it looked too dense, so thought to remove it.